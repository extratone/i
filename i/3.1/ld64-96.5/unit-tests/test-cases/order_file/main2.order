_main3
_aaa3
_main4



