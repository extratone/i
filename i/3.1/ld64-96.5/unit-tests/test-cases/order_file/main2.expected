_main3
_foo1
_aaa2
_bbb2
_ccc2
_aaa3
_bbb3
_ccc3
_aaa4
_main4
_main2
